package.json

```diff
@@ -108,6 +108,8 @@
     "typescript": "^4.0.3"
   },
   "dependencies": {
-    "clsx": "^1.1.0"
+    "@types/tinycolor2": "^1.4.2",
```

Comment:
I'd rather not add any more dependencies.
Is this really required? there are some color manipulation functions in material already (and reexported I think)